medusa/failed_processor.py

```diff
@@ -44,8 +44,8 @@ def process(self):
             parsed = NameParser().parse(releaseName)
         except (InvalidNameException, InvalidShowException):
             self.log(logger.WARNING, u'Not enough information to parse release name into a valid show. '
-                      u'Consider adding scene exceptions or improve naming for: {release}'.format
-                      (release=releaseName))
+                     u'Consider adding scene exceptions or improve naming for: {release}'.format
+                     (release=releaseName))
```

Comment:
Imo what we currently have is not too bad. Why remove the suggestion to improve the naming? It's a possible way to handle it. Maybe we could just add the shortened link in brackets after "Consider adding scene exceptions".